src/elements/Link/Link.tsx

```diff
@@ -24,7 +22,8 @@ export const Link = styled.a<LinkProps>`
     color: ${color("black100")};
   }
   &:focus {
-    border: 1px solid ${color("purple100")};
     color: ${props => (props.color ? color(props.color) : color("black100"))};
   }
+  ${space};
+  ${styledColor};
```

Comment:
We need those at the end otherwise they don't override CSS properties as expected.

Reply:
👍

Reply:
👍